lib/model/asset_assignment.php

```diff
@@ -120,20 +120,19 @@ private function fetch() {
 
 	/**
 	 * Generate a human readable title.
-	 * 
+	 *
 	 * Return name and, if available, the subtitle. Separated by a dash.
-	 * 
+	 *
 	 * @return string
 	 */
 	public function full_title() {
 		$t = $this->title;
-		
+
 		if ( $this->subtitle )
 			$t = $t . ' - ' . $this->subtitle;
-		
+
 		return $t;
 	}
 }
 
 AssetAssignment::property( 'image' );
-AssetAssignment::property( 'chapters' );
```

Comment:
Coming back to this PR.
Here you delete the option storage for chapters. Can you explain why you do this? I can't find a replacement.You seem to remove the whole functionality of choosing between manual entry and external assets.